packages/next/src/client/link.tsx

```diff
@@ -249,6 +255,50 @@ function formatStringOrUrl(urlObjOrString: UrlObject | string): string {
  */
 const Link = React.forwardRef<HTMLAnchorElement, LinkPropsReal>(
   function LinkComponent(props, forwardedRef) {
+    let children: React.ReactNode
```

Comment:
this block was just moved higher so I could use the isAppRouter variable for the dev warning